docs/api-docs/store-management/shipping/shipper-hq.md

```diff
@@ -74,7 +74,7 @@ subtitle: ""
 lineNumbers: true
 -->
 
-**Example Add ShipperHQ Metafield Response**  
+**Example add ShipperHQ metafield response**  
 `200 /POST https://api.bigcommerce.com/stores/{{store_hash}}/v3/catalog/products/{{product_id}}/metafields`

```

Comment:
I think we can remove this code since we are providing a link to the endpoint in Resources.

Reply:
Done